pom.xml

```diff
@@ -228,6 +229,11 @@
                 <artifactId>commons-math</artifactId>
                 <version>${commons-math.version}</version>
             </dependency>
+            <dependency>
+                <groupId>commons-collections</groupId>
+                <artifactId>commons-collections</artifactId>
+                <version>${commons-collections.version}</version>
+            </dependency>
```

Comment:
This version of commons collections is old and based on a very old version. We should instead use commons collections version 4 which is much more recent.
````
<dependency>
    <groupId>org.apache.commons</groupId>
    <artifactId>commons-collections4</artifactId>
    <version>4.4</version>
</dependency>
````
This requires a `4` to be added to import statements in a couple of files.